### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by mikeuri**

### CODE REVIEW

File: src/test/java/TestIsRetired.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        TRUE,
+        FALSE
+    }
+
+    @DataProvider
+    public Object[][] testDataMan() {
+        return new Object[][]{
+                {"Kelsey", "Gardner", 43, null, IsRetiredExpectedStatus.FALSE},
+                {"Christopher", "Glisson", 89, null, IsRetiredExpectedStatus.TRUE},
+                {"Winston", "Cox", 65, null, IsRetiredExpectedStatus.TRUE},
```
Comment (yurii-voronenko) [inline src/test/java/TestIsRetired.java:27]: AS I described in previous homework test boundary values 64, 65, 66

**Review by yurii-voronenko — APPROVED** [review]
LGTM
